examples/js/exporters/GLTFExporter.js

```diff
@@ -1028,6 +1004,73 @@ THREE.GLTFExporter.prototype = {
 
 			}
 
+			var forceIndices = options.forceIndices;
+			var isMultiMaterial = Array.isArray( mesh.material );
+
+			if ( ! forceIndices && geometry.index === null && isMultiMaterial ) {
+
+				// temporal workaround.
+				console.warn( 'THREE.GLTFExporter: Creating index for non-indexed multi-material mesh.' );
+				forceIndices = true;
+
+			}
+
+			var forcedIndex = false;
```

Comment:
minor: just to avoid confusing `forceIndices` and `forcedIndex`, could you rename this one to `didForceIndices` or something like that?

Reply:
Updated, thanks.